runtime/core.go

```diff
@@ -549,27 +550,34 @@ func Next(f *Frame, iter *Object) (*Object, *BaseException) {
 	return next.Fn(f, iter)
 }
 
-// Print implements the Python print statement. It calls str() on the given args
-// and outputs the results to stdout separated by spaces. Similar to the Python
-// print statement.
-func Print(f *Frame, args Args, nl bool) *BaseException {
-	// TODO: Support outputting to files other than stdout and softspace.
+// pyPrint encapsulates the logic of the Python print function.
+func pyPrint(f *Frame, args Args, sep, end string, file io.Writer) *BaseException {
```

Comment:
Please move this to the end of the file. I generally try to keep private helper functions at the end of the file. Also, maybe call it printImpl.